sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -0,0 +1,268 @@
+      </item>
+      <item>
+          <EFDB_Name>status</EFDB_Name>
+	  <Description>Status Code Telemetry</Description>
+          <IDL_Type>unsigned short</IDL_Type>
+          <Units>unitless</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+</SALTelemetry>
+<SALTelemetry>
+ <EFDB_Topic>ESS_temperature1Ch</EFDB_Topic>
+      <Description>Temperature Sensors (4-Channel) Serial to Ethernet Converter</Description>
+      <item>
+          <EFDB_Name>TemperatureC01</EFDB_Name>
+	  <Description>Temperature Sensor 1 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degrees</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+</SALTelemetry>
+<SALTelemetry>
+ <EFDB_Topic>ESS_temperature4Ch</EFDB_Topic>
+      <Description>Temperature Sensors (4-Channel) Serial to Ethernet Converter</Description>
+      <item>
+          <EFDB_Name>temperatureC01</EFDB_Name>
+	  <Description>Temperature Sensor 1 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degrees</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>temperatureC02</EFDB_Name>
+	  <Description>Temperature Sensor 2 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degrees</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>temperatureC03</EFDB_Name>
+	  <Description>Temperature Sensor 3 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degrees</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>temperatureC04</EFDB_Name>
+	  <Description>Temperature Sensor 4 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degrees</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+</SALTelemetry>
+<SALTelemetry>
+ <EFDB_Topic>ESS_temperature8Ch</EFDB_Topic>
+      <Description>Temperature Sensors (8-Channel) Serial to Ethernet Converter</Description>
+      <item>
+          <EFDB_Name>temperatureC01</EFDB_Name>
+	  <Description>Temperature Sensor 1 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degreesC</Units>
+          <Conversion></Conversion>
+          <Sensor_location></Sensor_location>
+          <Count>1</Count>
+      </item>
+      <item>
+          <EFDB_Name>temperatureC02</EFDB_Name>
+	  <Description>Temperature Sensor 2 Telemetry</Description>
+          <IDL_Type>float</IDL_Type>
+          <Units>degrees</Units>

```

Comment:
Cosnistency! Either degreesC or degrees, but not both. My preference is for degreesC